tests/main.rs

```diff
@@ -54,5 +54,5 @@ fn tests() -> TestTree {
 }
 
 fn main() {
-    default_main(Config::default(), tests())
+    default_main(Config::default(), tests());
```

Comment:
Are we returning non-zero value from `main` this way? From what I've read (e.g. [1] or [2]), `main` can or maybe even should return `Result`. It's unclear though whether it's better to return Result or `std::process::exit(non_zero)` elsewhere in the code, but returning Result seems to me a bit more explicit.

[1] https://stackoverflow.com/questions/24245276/why-does-rust-not-have-a-return-value-in-the-main-function-and-how-to-return-a
[2] https://doc.rust-lang.org/rust-by-example/error/result.html#using-result-in-main

Reply:
Excellent point, we should return non-zero status on test failure.

Reply:
Done; because this is a test for raclette, there are two expected failures here.